editor/plugins/sprite_frames_editor_plugin.h

```diff
@@ -115,7 +115,8 @@ class SpriteFramesEditor : public HSplitContainer {
 	SpinBox *frame_duration = nullptr;
 	ItemList *frame_list = nullptr;
 	bool loading_scene;
-	int sel;
+	Vector<int> selections;
```

Comment:
```suggestion
	Vector<int> selection;
```

or

```suggestion
	Vector<int> selected_frames;
```

Reply:
There is already a frames_selected variable, so I'll go with selection